components/widgets/menu/menu_items/menu_item_toggle_modal_redux.tsx

```diff
@@ -16,18 +16,18 @@ type Props = {
     dialogType: React.ComponentType<any>;
     dialogProps?: Dictionary<any>;
     extraText?: string;
-    text?: string;
-    accessibilityLabel?: string;
+    text?: React.ReactNode;
+    ariaLabel?: string;
```

Comment:
@michelengelen This breaks accessibility in several places. Like in the screenshot, it only reflect ` dialog` where previously it was `Custom Status dialog`.

<img width="760" alt="Screen Shot 2021-09-29 at 2 23 56 PM" src="https://user-images.githubusercontent.com/5334504/135214705-cacd2de2-2838-4ee4-b0ad-a4acf5fe7c50.png">